server/src/services/metadata.service.ts

```diff
@@ -334,6 +334,18 @@ export class MetadataService extends BaseService {
     return JobStatus.SUCCESS;
   }
 
+  private durationToString(duration?: string | number | ExifDuration) {
+    if (!duration) {
+      return null;
+    }
+    if (!duration.hasOwnProperty('Value')) {
+      return duration?.toString();
```

Comment:
Have you confirmed that returning `undefined` instead of `null` still works as expected? Maybe if there was a duration previously, but none after rerunning metadata extraction, it might not get updated correctly now.

Reply:
No, good catch. I'll fix this. I'm not used to TypeScript (and I don't think I will ever get used to having null, 0, false, false-y, undefined to pick from).

Reply:
Ah, actually duration is non-null-non-undefined at this point, so it's not an issue. I can remove the "?".